### PR TITLE
payment link: reword claim/reclaim to accept/cancel

### LINKED ISSUE #571: Improve language around payment links
“Claim”/“Reclaim” is a bit confusing for payment links semantics unless you understand the underlying details of how the link works. 

Suggestion from dwr:
- Claim -> Accept
- Reclaim -> Cancel

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
Closes #571 

Based on #577 , merge that first.

### CODE REVIEW

File: apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx [hunk 1]
```diff
@@ -253,7 +253,7 @@ function NoteDisplayInner({
         if (netRecv === 0) {
           return `Gas too high to claim`;
         } else if (isOwnSentNote) {
-          return `Cancel this link, reclaiming ${netDollarsReceivedStr}`;
+          return `Cancel this link, recollecting ${netDollarsReceivedStr}`;
```
Comment (dcposch) [inline apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx:256]: i think "reclaiming" is fine here, recollect means remember

File: packages/daimo-api/test/pushNotifier.test.ts [hunk 2]
```diff
@@ -206,7 +206,7 @@ test("PushNotifier", async () => {
 
     assert.strictEqual(output.length, 1);
     assert.deepStrictEqual(output[0].to, ["pushTokenAlice"]);
-    assert.strictEqual(output[0].title, "$4.20 claimed");
+    assert.strictEqual(output[0].title, "Recollected $4.20");
```
Comment (dcposch) [inline packages/daimo-api/test/pushNotifier.test.ts:209]: maybe "Reclaimed $4.20"? this is after you cancel your own link

**Review by dcposch — APPROVED** [review]
LGTM after comments

File: apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx (hunk 1 above)
Comment (nalinbhardwaj) [reply apps/daimo-mobile/src/view/screen/link/NoteScreen.tsx]: done

File: packages/daimo-api/test/pushNotifier.test.ts (hunk 2 above)
Comment (nalinbhardwaj) [reply packages/daimo-api/test/pushNotifier.test.ts]: done

**Pull request merged**